metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/params/params_linux.py

```diff
@@ -76,6 +81,9 @@
 global_json_template = config['configurations']['metron-env']['global-json']
 global_properties_template = config['configurations']['metron-env']['elasticsearch-properties']
 
+# Java
+java64_home = config['hostLevelParams']['java_home']
```

Comment:
Is this necessary?  This property is also defined in params.py.

Reply:
Nope, I dropped it and the reference to it is changed to just java_home